Obedient-Lobsters/BangazonAPI
# Js Product Controller

# Description
This is a basic Product controller that allows one to Get a single product back, or Get all products back.
Along with that, a user can Post products to the database, edit (Put) products, and delete products.

 Fixes # (2)
 ## Type of change
 Please delete options that are not relevant.

 # Testing Instructions
Start the program by cd'ing into the BangazonAPI and using the command dotnet run. Once the program is running, open up the Postman desktop app and run the following commands:

select GET then paste localhost:5000/Product into the field and click send. The result should be an array of all the Products in the database.

select GET then paste localhost:5000/Product/2 or any other number that showed up in the previous query as ProductId and click send. The result should be only that object of the specified Product.

select POST, then paste localhost:5000/Product into the field, then click Body underneath the field, then select raw, and then paste this snippet or make one similar
`{
	"price": 43.34,
	"title": 'details',
	"description": 'more details',
	"customerId": 2,
	"quantity": 3,
	"productTypeId": 2
}`
then click send. The result should be the new Product you made.

select PUT then paste localhost:5000/Product/2 or any other ProductId then do the same as the POST example and change the values then click send. You should get nothing back from this. When you run the GET query the computer you specified in your PUT query should show the updated, edited information you gave it.

select DELETE then paste localhost:5000/Product/2 or any other ProductId then click send. You should get nothing back from this. When you run the GET query the computer you specified in your DELETE query should no longer exist.

 # Checklist:
